Presentation/MemoryUtils.swift

```diff
@@ -34,6 +34,8 @@ public let enabledDisplayAlertOnMemoryLeaksKey = "enabledDisplayAlertOnMemoryLea
 
             func onLeak() {
                 log(.didLeak(.init(vcPresentationDescription), from: .init(presentationDescription)))
+                let memoryLeak = UIVCMemoryLeak(name: self.presentationTitle)
```

Comment:
So as I understand we store the title of controller where the leak had occurred. But how we going to use that data?

Reply:
Once we got stored files with names of view controllers after running UI tests we can process it at the end on CI.